source/features/pr-commit-lines-changed.tsx

```diff
@@ -43,6 +43,7 @@ async function init(): Promise<void> {
 features.add({
 	id: __featureName__,
 	description: 'Add line changes on PR commits.',
```

Comment:
I wonder if this should be
```suggestion
	description: 'Adds diff stats on PR commits.',
```

Reply:
Agree